api/domain/customersDomain.js

```diff
@@ -38,5 +38,29 @@ exports.delete = async (customerId) => {
         throw new Error('No customer deleted.');
     }
 
+    return affectedRows;
+};
+
+exports.getAll = async (customers) => {
+    let affectedRows = await customerRepo.getAll(customers);
```

Comment:
affectedRows is a term used for rows in a table in db that have been somehow affected - for example rows that have been updated or deleted. In terms of getting data in doesn't make sense